apps/dav/lib/CardDAV/CardDavBackend.php

```diff
@@ -1062,6 +1063,7 @@ private function searchByAddressBookIds(array $addressBookIds,
 											array $searchProperties,
 											array $options = []): array {
 		$escapePattern = !\array_key_exists('escape_like_param', $options) || $options['escape_like_param'] !== false;
+		$useWildcards = !\array_key_exists('wildcard', $options) || $options['wildcard'] !== false;
```

Comment:
Or add it to this condition

```suggestion
		$useWildcards = !(array_key_exists('wildcard', $options) && $options['wildcard'] === true) || (array_key_exists('strict_search', $options) && $options['strict_search'] === true);
```

Reply:
Or that, yeah